lib/cdo/aws/resource_management/test/test_ec2_instance_for_deletion.rb

```diff
@@ -0,0 +1,74 @@
+require_relative '../../../../../shared/test/common_test_helper'
+require_relative '../ec2_instance_for_deletion'
+
+class TestEC2InstanceForDeletion < Minitest::Test
+  class MockEC2Instance
+    attr_accessor :tags
+
+    def initialize(api_disabled, tags)
+      @tags = tags
+      @api_disabled = api_disabled
+    end
+
+    def describe_attribute(attribute:)
+      aws_response = OpenStruct.new(value: @api_disabled)
+      OpenStruct.new(disable_api_termination: aws_response)
+    end
+
+    def stack_name
+      @tags.find {|tag| tag.key == 'aws:cloudformation:stack-name'}&.value
+    end
+
+    def stop_time_in_seconds
+      stopped_at_tag = @tags.find {|tag| tag.key == 'deletionTime'}
+      return nil if stopped_at_tag.nil?
+      DateTime.parse(stopped_at_tag.value)
+    end
+  end
+
+  TOO_SOON_TIME_TO_DELETE = Time.now - (5 * 24 * 60 * 60)
+  READY_TO_DELETE_TIME_IN_SECS = Time.now - (7 * 24 * 60 * 60)
```

Comment:
could these be named more consistently? I'd also long to see some comments explaining the significance of these values.